grammars/go.cson

```diff
@@ -162,19 +169,37 @@
         "match": "(\\+|&|\\+=|&=|&&|==|!=|\\-|\\||\\-=|\\|=|\\|\\||<|<=|\\*|\\^|\\*=|\\^=|<\\-|>|>=|/|<<|/=|<<=|\\+\\+|=|:=|;|%|>>|%=|>>=|\\-\\-|!|\\.\\.\\.|&\\^|&\\^=)"
       }
       {
-        "name": "keyword.operator.bracket.go",
-        "match": "(\\(|\\)|\\[|\\]|\\{|\\})"
+        'match': '\\{|\\}'
```

Comment:
Since braces aren't listed as keywords in the spec, I don't think they should have a scope name with `keyword` in them.

The braces are described in the section on [Operators and Delimiters](https://golang.org/ref/spec#Operators_and_Delimiters), but they are never mentioned in the [Operators](https://golang.org/ref/spec#Operators) section. I concluded that they are are not operators, and must therfore be delimiters.

The reason for changing the scopes is to better align braces with how other packages (C++, C#, JavaScript, CSS, Perl, Shell, Python etc.) treat braces. The advantage of aligning with them is that a larger number of syntax highlighting themes will work well equally with Go's scopes. From these few examples of `{` in other packages, I concluded that `meta`, `punctuation` and `scope` are good scope names for curly braces, besides `brace.curly` of course.
<img width="444" alt="screen shot 2015-07-24 at 12 10 49" src="https://cloud.githubusercontent.com/assets/703602/8872423/2c1ed132-31fd-11e5-97e7-2f44aea2e2e6.png">

What is your opinion on `meta.scope.brace.curly.go`?